src/templates/app/user/statement.jsx

```diff
@@ -1,30 +1,34 @@
 import React from 'react';
 import Loading from '../../_common/components/loading.jsx';
 
+const AccountStatistics = () => (
+    <div id='account_statistics' className='gr-row invisible'>
+        <div className='gr-3 gr-12-m'>
+            <p className='title'>{it.L('Account statistics')}</p>
+        </div>
+        <div className='gr-3 gr-12-m'>
+            <span className='hint'>{it.L('Total deposits')}</span>
+            <p id='total_deposits' />
+        </div>
+        <div className='gr-3 gr-12-m'>
+            <span className='hint'>{it.L('Total withdrawals')}</span>
+            <p id='total_withdrawals' />
+        </div>
+        <div className='gr-3 gr-12-m fill-bg-color'>
+            <span className='hint'>{it.L('Net deposit')}</span>
+            <p id='net_deposit' />
+        </div>
```

Comment:
I meant small blocks for each div since they are repeating 😅 something like 
```
<div className={`gr-3 gr-12-m ${className}`}>
    <span className='hint'>{text}</span>
    <p id={id} />
</div>
```